core-developers/developers.csv

```diff
@@ -1,180 +1,180 @@
-Erlend Egeberg Aasland,erlend-aasland,2022-05-05,,

```

Comment:
Did the line endings change or is it just git getting confused by the move?
This file is generated, so it shouldn't be changed.

In addition, I'm planning to move this in an `include` directory as part of #884.

Reply:
> just git getting confused by the move?

This, the line endings shouldn't've changed.

A

Reply:
Reverted.

A

Reply:
Thanks for reverting!  As I mentioned that file is generated, so reverting the line endings change is the right thing to do, or the next update to the file will result in a big diff that will revert them all.